WCSim.mac

```diff
@@ -26,8 +28,11 @@
 
 #Use mPMTs settings (uncomment/delete the above)
 #/WCSim/WCgeom nuPRISM_mPMT
-#/WCSim/WCgeom nuPRISMBeamTest_mPMT
-/WCSim/WCgeom nuPRISMShort_mPMT
+#/WCSim/WCgeom nuPRISMBeamTest_mPMT ## this is 18c5r from the original design
```

Comment:
Please create a new `.mac` instead, like `WCSim/macros/NuPRISM.mac`, for WCTE specific run configuration. 

Other general improvements can stay in `WCSim.mac`. And actually, should consider having a base `.mac` that specific `.mac`s can call.

Reply:
Have updated to nuPRISMBeamTest.mac in my repo - will do another pull request with all changes later(?)

Reply:
pushing to this branch should update this PR automatically

Reply:
this actually removes `WCSim.mac`. can you add it as a new file instead?

Reply:
done